synthtool/gcp/common.py

```diff
@@ -252,6 +252,10 @@ def py_library(self, **kwargs) -> Path:
         ):
             self.excludes += ["docs/index.rst"]
 
+        # Add kwargs to signal that upgrading.rst should be included in docs/index.rst if it exists
+        if Path("docs/upgrading.rst").exists():
```

Comment:
I believe we used a MD file in most repos: https://github.com/googleapis/python-access-approval/blob/main/UPGRADING.md